### PR TITLE
Python.Launcher.3.12 version 3.12.0

### LINKED ISSUE #121195: [Package Request]: Python py.exe launcher
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://www.python.org/downloads/
Publisher: Python Software Foundation
Package Name: Python launcher
Description: Provides the 'py.exe' launcher
Package Version: 3.11.5
Installer URL: https://www.python.org/ftp/python/3.11.5/win32/launcher.msi

This is normally installed as part of CPython, but can also be installed separately (on purpose, just not widely advertised). The launcher is able to detect Windows Store installs (and to install them via winget), as well as any other properly registered Python distribution, so it can be very useful for people who use those installs but do not want a regular CPython install.

While the version number matches the version of CPython it is bundled with, it does not install in parallel like CPython does, and in general the latest release version is all that should be made available (e.g. when 3.12.0 releases in a couple of weeks, nobody should need any 3.11 versions of the launcher anymore). The URL can be inferred by taking any CPython release URL (the ".exe" file) and replacing the filename with `win32/launcher.msi`. It is only ever released as a 32-bit executable.

The installer defaults to per-user, which is generally fine, but if it is run elevated and `ALLUSERS=1` passed to msiexec it will install for all users.

### PR DESCRIPTION
**Pull request by Dragon1573**
Resolve https://github.com/microsoft/winget-pkgs/issues/121195 .

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?
- [x] Have you tested your manifest in Windows Sandbox with `Tools/SandboxTest.ps1 -Manifest <path>`?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/121756)

### CODE REVIEW

**Comment by zooba** [comment]
LGTM :shipit:

File: manifests/p/Python/Launcher/3/12/3.12.0/Python.Launcher.3.12.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+- pyzw
+ProductCode: '{3182A195-B671-44A8-B0C7-7876B916BA5A}'
+ElevationRequirement: elevatesSelf
+Installers:
+- Architecture: x86
+  Scope: user
+  InstallerUrl: https://www.python.org/ftp/python/3.12.0/win32/launcher.msi
+  InstallerSha256: 3A0DAA56286B02546438D24F158A334F4936B5EB027BECCDA60EAA0B39C075BA
+  InstallerSwitches:
+    Custom: ALLUSERS=0
```
Comment (mdanish-kh) [inline manifests/p/Python/Launcher/3/12/3.12.0/Python.Launcher.3.12.installer.yaml:29]:
```suggestion
```
Not required. The installer installs in user mode without these switches

File: manifests/p/Python/Launcher/3/12/3.12.0/Python.Launcher.3.12.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+FileExtensions:
+- py
+- pyc
+- pyd
+- pyo
+- pyw
+- pyz
+- pyzw
+ProductCode: '{3182A195-B671-44A8-B0C7-7876B916BA5A}'
+ElevationRequirement: elevatesSelf
```
Comment (mdanish-kh) [inline manifests/p/Python/Launcher/3/12/3.12.0/Python.Launcher.3.12.installer.yaml:22]:
```suggestion
```

The installer does not prompt for elevation if being installed in user mode.

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

File: manifests/p/Python/Launcher/3/12/3.12.0/Python.Launcher.3.12.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+Tags:
+- language
+- programming
+- programming-language
+- python
+- python3
+- script
+Documentations:
+- DocumentLabel: Documentation
+  DocumentUrl: https://www.python.org/doc/
```
Comment (mdanish-kh) [inline manifests/p/Python/Launcher/3/12/3.12.0/Python.Launcher.3.12.locale.en-US.yaml:33]:
```suggestion
- DocumentLabel: Python Launcher for Windows
  DocumentUrl: https://docs.python.org/3/using/windows.html#launcher
```

File: manifests/p/Python/Launcher/3/12/3.12.0/Python.Launcher.3.12.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,37 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Python.Launcher.3.12
```
[the remark this answers is not in the document]
Comment (Dragon1573) [reply manifests/p/Python/Launcher/3/12/3.12.0/Python.Launcher.3.12.installer.yaml:4]: Okay, thank you! I'll change manifests directory on my local device.

**Review by mdanish-kh — APPROVED** [review]
Since the path has been modified, this needs a waiver again to get merged

cc @stephengillie

**Pull request merged**